i18n/Spanish.json

```diff
@@ -0,0 +1 @@
+English.json
```

Comment:
I've added a project in POEditor and tagged Luis there, so I hope we can get the translations soon.